### DISCUSSION TITLE
Is audio prompt needed for LJSpeech model ?

### DISCUSSION THREAD

**Discussion by theodorblackbird** [lifeiteng/valle#167] [Q&A] [answered]
Hey,
In your demo page you show a smaller LJSpeech model. You keep using an audio prompt for generation.
Is the model able to generate without any audio prompt since it has only seen one speaker ? 
Thanks

**Comment by lifeiteng** [comment, marked as answer]
I don't test the model in this way. You can try, it makes sense.
